export/notebook.tex

```diff
@@ -58,9 +40,15 @@ \chapter{Numerical}
 [[kactl/numerical/MatrixInverse.h]]
 %[[kactl/numerical/Tridiagonal.h]]
 [[kactl/numerical/FFT.h]]
+[[trinerdi/numerical/ntt.cpp]]
```

Comment:
KACTL puts NTT in number-theory. I agree with putting it in the same section as FFT, but then it would be a good idea to either comment out KACTL's implementation or move it here.